lang/cs/.nuget/NuGet.Config

```diff
@@ -21,11 +21,7 @@ under the License.
   <solution>
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
-  <packageSources>
```

Comment:
Yes. Was auto added by VS2015 when I was trying to run tests. I am reverting this change.